package.json

```diff
@@ -50,7 +50,8 @@
     "install:server": "yarn workspaces focus @actual-app/sync-server --production",
     "typecheck": "yarn tsc --incremental && tsc-strict",
     "jq": "./node_modules/node-jq/bin/jq",
-    "prepare": "husky"
+    "prepare": "husky",
+    "postinstall": "patch-package"
```

Comment:
FYI: you should be able to do `yarn patch adm-zip` without needing to bring in another dependency here. :)

Reply:
Much nicer, thanks. I didn't know that.

Fixed now